.pnpmfile.cjs

```diff
@@ -39,6 +39,10 @@ function readPackage(pkg, context) {
       "@types/react": { optional: true },
     };
   }
+  
+  if(pkg.name === "@casperlabs/ts-results") {
+    pkg.dependencies['tslib'] = '^1.14.1';
+  }
```

Comment:
Can you please explain the need for this?

Reply:
This is one of really old problem and have been fixed in the newer commits.

More info: https://github.com/casper-ecosystem/casper-js-sdk/pull/212